### PR TITLE
Update frontend.php (lang ES)

### PR DESCRIPTION
**Pull request by julf0**
Add new lines from the original lang

@jezevec10 check the length of the strings **first.**

### CODE REVIEW

File: es/frontend.php [hunk 1]
```diff
@@ -98,4 +103,49 @@
...
+    'newPB' => 'TU NUEVA MARCA PERSONAL',
+    'firstPB' => 'Este fue tu primer juego. Haz otra marca personal para hacer un seguimiento de las estadisticas de mejora.',
+    'infoPB' => 'Tu anterior record era {prevPB} logrado {prevAgo}. La mejora es {PBdiff}.',
+    'daysAgo' => 'días atras',
+
+    'raceFin' => 'LA CARRERA TERMINO',
+    'raceFinInfo' => 'Puedes completar la carrera, pero la proxima ronda puede empezar en cualquier momento.',
+
+    'notTSD' => 'NO TSD',
+    'notTSDInfo' => 'Solamente es permitido el T-Spin Double',
```
Comment (jezevec10) [inline es/frontend.php]:
```suggestion
    'notTSDInfo' => 'Solamente es permitido<br>el T-Spin Double',
```
accept suggestion

File: es/frontend.php [hunk 2]
```diff
@@ -98,4 +103,49 @@
...
+
+    //Room info detail
+    'noPlayers' => 'Sin jugadores',
+    'cntMore' => '{cnt} más',          //e.g. +10 more
+    'cntGuests' => '{cnt} invitados',      //e.g. +10 guests
+    'cntSpec' => '{cnt} espectadores',    //e.g. +10 spectating
+    'joinPossible' => 'Puedes ingresar',
+    'notEligible' => 'No personalizable',
+    'gTimeShort' => 'T.juego',
+    'on' => 'Habilitado',
```
Comment (jezevec10) [inline es/frontend.php]: Too long, try max 4 characters

File: es/frontend.php [hunk 3]
```diff
@@ -98,4 +103,49 @@
...
+    'off' => 'Deshabilitado',
```
Comment (jezevec10) [inline es/frontend.php]: Too long, try max 4 characters

File: es/frontend.php [hunk 4]
```diff
@@ -98,4 +103,49 @@
...
+	
+    'oops' => 'Uf!',
+    'chatNA' => 'El chat publico no esta habilitado para invitados o usuarios con menos de {chReq} horas de juego.',
+    'leMore' => 'Aprende más',
+
+    'connLimit' => 'Las conexiones máximas para esta IP ha sido alcanzada. Si necesitas incrementar este limite contactanos por Discord',
+    'idleDC' => 'Desconectado por inactividad! La zona de espectadores esta llena.',
+    'RLreach' => 'Limite de velocidad alcanzado.',
+    'ban1' => 'El acceso a los juegos ha sido restringido permanentemente. Todavía puedes jugar los modos de un solo jugador.',
+    'ban2' => 'This user account is BANNED. Relogin to the website for more information.',
```
Comment (jezevec10) [inline es/frontend.php]: English version can be removed, or provide translation

**Review by jezevec10 — CHANGES_REQUESTED** [review]

File: es/frontend.php (hunk 2 above)
Comment (julf0) [reply es/frontend.php]: Fixed on into 'sí'

File: es/frontend.php (hunk 3 above)
Comment (julf0) [reply es/frontend.php]: Fixed off into 'no'

File: es/frontend.php (hunk 4 above)
Comment (julf0) [reply es/frontend.php]: My fault, I did not see it, **fixed**

**Review by jezevec10 — APPROVED** [review]

**Pull request merged**